test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
This test now uses an empty string image field, isn't that what you want
tested?

Ben Parees | OpenShift
On Jun 25, 2016 6:30 AM, "Michail Kargakis" <notifications@github.com>
wrote:

> In test/extended/testdata/test-imagesource-build.yaml
> <https://github.com/openshift/origin/pull/9535#discussion_r68487273>:
>
> > @@ -89,7 +89,7 @@ items:
> >            deploymentconfig: imagesourceapp
> >        spec:
> >          containers:
> > -        - image: imagesourceapp
> > +        - image: " "
>
> @gabemontero <https://github.com/gabemontero> can you please add an
> extended test in the deployment suite that exercises the missing image name?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9535/files/4896163ca46c222813833ff9b81f4c663cba35f4#r68487273>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEvl3lHEk2hSCExTOOLe-qxHVF8dXNEHks5qPQMkgaJpZM4I9cEy>
> .
>